### ISSUE TITLE
false positive: missing-test-assertion reported for `clojure.test/are`

### ISSUE THREAD

**Issue by gordonsyme** [borkdude/clj-kondo#413]
**version**

2019.07.31-alpha

**platform**

macOS High Sierra

**problem**

Sole use of `clojure.test/are` in a test is reported as a missing test assertion

**repro**

```clojure
(ns false-positive.reproduction-test
  (:require [clojure.test :refer (are deftest)]))

(deftest missing-test-assertion-false-positive
  (are [v expected?] (expected? (contains? #{:bar :baz} v))
       :foo false?
       :bar true?
       :baz true?))
```

Invoked with `lein clj-kondo --cache --lint "$(lein classpath)"`

**expected behavior**

The test is not flagged for a missing test assertion

**actual behavior**

clj-kondo emits this warning:
```
/Users/gordon/Development/foo/test/false_positive/reproduction_test.clj::: warning: missing test assertion
```

The line- and column-numbers are also missing from the warning.

**Comment by borkdude** [comment]
@gordonsyme If you want, you can test with this binary:

https://5345-176829714-gh.circle-artifacts.com/0/release/clj-kondo-2019.08.01-alpha-SNAPSHOT-macos-amd64.zip

**Comment by borkdude** [comment]
Fixed with https://github.com/borkdude/clj-kondo/commit/71e9fcaaf82749e67e6160566245698fc6fbaaaf

**Comment by gordonsyme** [comment]
Thanks! That fixes the test assertion warnings.

There's a new `recur argument count mismatch` error showing up with that binary. It's in a slightly hairy bit of code using `core.async` within `loop`. I'll try to get a minimal reproduction of it.

**Comment by borkdude** [comment]
@gordonsyme Thanks! Please post the `recur` issue in a new issue.
